src/types.ts

```diff
@@ -98,45 +98,35 @@ export interface Attribute {
 
 export interface CodeableConcept {
```

Comment:
In the test classes I used various actual JSON result sets pulled back from my local dev server and some didn't match the typescript definitions, so I fixed so they were consistent... mainly there were main fields marked in this file as required/mandatory that weren't actually mandatory and not present on the test data.